docs/source/api/pystiche_papers.ulyanov_et_al_2016.rst

```diff
@@ -1,6 +1,103 @@
+``impl_params`` as in the following table, the appropriate parameters will be used:
+
+  - ``master``: The parameters specified in the implementation branch
+    `master <https://github.com/pmeier/texture_nets/tree/master>`_.
+  - ``texture_nets_v1``: The parameters specified in the implementation branch
+    `texture_nets_v1 <https://github.com/pmeier/texture_nets/tree/texture_nets_v1>`_.
+  - ``"paper"``: The parameters specified in the papers. Here ``instance_norm`` is only
+    used to use either :class:`~torch.nn.InstanceNorm2d` or
+    :class:`~torch.nn.BatchNorm2d`.
+
++-----------------+-----------+------------------------------------+
+|                 |           |        ``instance_norm``           |
++-----------------+-----------+-------------+----------------------+
+|                 |           |   ``True``  |       ``False``      |
++-----------------+-----------+-------------+----------------------+
+| ``impl_params`` | ``True``  | ``master``  | ``texture_nets_v1``  |
+|                 +-----------+-------------+----------------------+
+|                 | ``False`` | ``"paper"`` |      ``"paper"``     |
++-----------------+-----------+-------------+----------------------+
+
+.. _table-hyperparameters-ulyanov_et_al_2016:
+
+The following table provides an overview of the parameters:
+
+- the ``num_epochs`` of the :func:`~pystiche_papers.ulyanov_et_al_2016.training`,
+
+- the ``lr`` the learning rate of the
+  :func:`~pystiche_papers.ulyanov_et_al_2016.optimizer`,
+
+- the ``num_batches`` and ``batch_size`` of the
+  :func:`~pystiche_papers.ulyanov_et_al_2016.image_loader`,
+
+- the ``score_weight`` for
+
+  - the :func:`~pystiche_papers.ulyanov_et_al_2016.content_loss`,
+  - the :func:`~pystiche_papers.ulyanov_et_al_2016.style_loss`, as well as
+
+- the ``layers`` for
+
+  - the :func:`~pystiche_papers.ulyanov_et_al_2016.style_loss`.
+
+The ``layers`` have either ``4 layers`` ``("relu1_1", "relu2_1", "relu3_1", "relu4_1")``
+or ``5 Layer`` ``("relu1_1", "relu2_1", "relu3_1", "relu4_1", "relu5_1")``.
+
+
++-------------------+---------------+--------------+-----------------------+
+|   Parameter       |  ``"paper"``  |  ``master``  |  ``texture_nets_v1``  |
++===================+===============+==============+=======================+
+|  ``num_epochs``   |      10       |      10      |           25          |
++-------------------+---------------+--------------+-----------------------+
+|     ``lr``        |     1e-1      |    1e-3      |         1e-1          |
++-------------------+---------------+--------------+-----------------------+
+|  ``num_batches``  |     200       |     2000     |          300          |
++-------------------+---------------+--------------+-----------------------+
+|  ``batch_size``   |      16       |       1      |           4           |
++-------------------+---------------+--------------+-----------------------+
+| ``content_loss``  |               |              |                       |
```

Comment:
I don't think we should have sections within this table. Maybe `content_score_weight`? Same for the `style_loss` below.